src/components/howTos.stories.mdx

```diff
@@ -36,12 +36,11 @@ const App = () => {
 
 ## Enabling video and audio stream quality indicator
 
-You can use useMediaStreamMetrics hook to get video and audio stream quality metrics
-An [example](https://github.com/aws/amazon-chime-sdk-component-library-react/blob/de22a537d6437073f72d69da30f1703ef4cf40b3/demo/meeting/src/containers/VideoStreamMetrics/index.tsx#L27) of using the hook to show video quality metrics for every attendee in the meeting roster.
+You can use `useMediaStreamMetrics` hook to get video and audio stream quality metrics.
 
-An [example](https://github.com/aws/amazon-chime-sdk-component-library-react/blob/de22a537d6437073f72d69da30f1703ef4cf40b3/demo/meeting/src/containers/LocalMediaStreamMetrics/index.tsx#L22-L28) of using the hook to show video and audio quality metrics for local attendee.
+You can find examples and usage of the hook from `useMediaStreamMetrics` storybook documentation.
```

Comment:
Thanks for fixing this, can we add a link to the hook's storybook documentation here?